cfgov/unprocessed/css/brand-palette.less

```diff
@@ -4,69 +4,143 @@
 
-@green:             #2CB34A;
-@green-midtone:     #ADDC91;
-@green-tint:        #DBEDD4;
+// Green family
 
-@black:             #101820;
-
-
-// Background colors:
-
-@darkgray:          #43484E;
-
-@gray:              #75787B;
-@gray-80:           #919395;
-@gray-50:           #BABBBD;
-@gray-40:           #b4b5b6; // Close to what was known as "Gray 50"
-@gray-20:           #E3E4E5;
-@gray-10:           #F1F2F2;
-@gray-5:            #F8F8F8;
-
-
-// Secondary colors:
-
-@redorange:         #D12124;
-@redorange-80:      #DA6750;
-@redorange-50:      #E8A091;
-@redorange-20:      #F6D9D3;
-
-@gold:              #FF9E1B;
-@gold-80:           #FFB149;
-@gold-50:           #FFCE8D;
-@gold-20:           #FFECD1;
-
-@neutral:           #796E65;
-@neutral-80:        #948B84;
-@neutral-50:        #BCB6B2;
-@neutral-20:        #E4E2E0;
-
-@navy:              #002D72;
-@navy-80:           #33578E;
-@navy-50:           #7F96B8;
-@navy-20:           #CCD5E3;
-
-@pacific:           #0072CE;
-@pacific-80:        #328ED8;
-@pacific-50:        #7FB8E6;
-@pacific-20:        #CCE3F5;
-
-@teal:              #005E5D;
-@teal-80:           #337E7D;
-@teal-50:           #7FAEAE;
-@teal-20:           #CCDFDF;
-
-
-// Unofficial colors not part of the official palette,
-// but are in use for easier coding semantics or UX needs.
-
-// Active state of destructive action buttons.
-@dark-redorange:    #9C301B;
-
-@white:             #FFF;
+@dark-green:   #1e9642;
+@green:        #20aa3f; // Primary brand green color
+@green-80:     #66c368;
+@green-60:     #addc91; // Formerly known as "Green: Midtone"
+@green-40:     #c7e5b3;
+@green-20:     #e2efd8;
+
+
+// Teal family
+
+@dark-teal:    #005e5d; // Formerly known as regular ol' "Teal"
+@teal:         #257675; // Close to what was previously called "Teal 80"
+@teal-80:      #579695;
+@teal-60:      #89b6b5;
+@teal-40:      #b4d2d1;
+@teal-20:      #d4e7e6;
+
+
+// Pacific family
+
+@dark-pacific: #0050b4;
+@pacific:      #0072ce;
+@pacific-80:   #4497dc;
+@pacific-60:   #7eb7e8;
+@pacific-40:   #afd2f2;
+@pacific-20:   #d6e8fa;
+
+
+// Navy family
+
+@dark-navy:    #002d72; // Formerly known as regular ol' "Navy"
+@navy:         #254b87; // Close to what was previously called "Navy 80"
+@navy-80:      #5674a3;
+@navy-60:      #889cc0;
+@navy-40:      #b3c0d9;
+@navy-20:      #d3daeb;
+
+
+// Red family (formerly known as "Red Orange")
+
+@dark-red:     #b63014;
+@red:          #d14124;
+@red-80:       #dd735d;
+@red-60:       #e79e8e;
+@red-40:       #f0c3b8;
+@red-20:       #f7e0d9;
+
+
+// Gold family
+
+@dark-gold:    #dc731c;
+@gold:         #ff9e1b;
+@gold-80:      #ffb858;
+@gold-60:      #ffce8d;
+@gold-40:      #ffe1b9;
+@gold-20:      #fff0dd;
+
+
+
+// Neutral family
+
+@dark-neutral: #63574e;
+@neutral:      #796e65;
+@neutral-80:   #948b84;
+@neutral-60:   #b5aca5;
+@neutral-40:   #cdc7c2;
+@neutral-20:   #e3e2df;
+
+
+// Gray family
+
+@black:        #101820; // Also known as "CFPB Black"
```

Comment:
This may be to make the black used appear more consistent across screen and print and to visibly look like consistent gradient steps through the grays to black. Black can be created via a mix of RGB, a mix of CMYK, or as 0% C 0% M 0% Y 100% K. So when converting 100% black in CMYK to black in RGB the result is likely not `#000`. See https://github.com/cfpb/design-manual/issues/327#issuecomment-140925068
Since it's not `#000`, the `@black` variable makes it easier to work with. `@white` is provided for consistency and to handle if white were to be tinted to an off-white in the future. 

At any rate, we're kind of the end users of these names since they're decided in the design manual and https://github.com/cfpb/generator-cf/blob/master/app/templates/src/static/css/brand-palette.less, so would need to be changed there.